src/patternfly/demos/BasicForms/examples/index.js

```diff
@@ -11,12 +11,13 @@ import docs from '../docs/code.md';
 
 export const Docs = docs;
 
-export default (props) => {
+export default props => {
   const simpleFormDemoExample = SimpleFormDemoExample();
   const horizontalFormDemoExample = HorizontalFormDemoExample();
+  const headingText = 'Basic Forms Demo';
```

Comment:
should be sentence case

```suggestion
  const headingText = 'Basic forms demo';
```

Reply:
Done.